docs/wfforward.md

```diff
@@ -0,0 +1,1117 @@
+    MutationMetaData(origin=284, pos=0.86918038199988279)
+    MutationMetaData(origin=923, pos=0.86932622547380545)
+
+
+The meta-data positions match the positions in the site table (phew!).  The times when the mutations arose are encoded forwards in time.  We simulated for 1,000 generations, meaning that the allele ages are 1,001 - origin, and we can check that the mutation age is greater than the node time it is found on and less than the node time of its parental node:
+
+
+```python
+for mi in m:
+    x = pickle.loads(mi.metadata)
+    age = 1001 - x.origin
+    node_time = n[mi.node].time
+    node_index = np.where(e.child == mi.node)[0]
+    pnode_time = None
+    if len(node_index) != 0:
+        pnode_time = n[e[node_index[0]].parent].time
+        assert(age < pnode_time)
+    print(x.pos, node_time, age, pnode_time)
+    assert(age > node_time)
+```
+
+    0.221962833876 311.0 831 None
+    0.825857514891 74.0 142 163.0
+    0.869180382 311.0 717 None
+    0.869326225474 -0.0 78 163.0
+
+
+**Detail:** the simplistic searching for parents of nodes used above only works because we simulated without recombination.  With recombination, a parent can have multiple segments leading to a child, meaning multiple rows in an `msprime.EdgeTable`.  For such cases, you have to search both by position and by node id.
+### Summary and comments
+
+For many standard modeling scenarios, it is straightforward to start the forward simulation with a set of nodes and edges generated by a coalescent simulation.  However, it is important to recognize that this approach does not easily generalize.  If your forward simulation involves complex demographic and/or life history scenarios, then it is very likely that its effective population size is quite different from that assumed by Kingman's coalescent or that an $N_e$ simply does not exist for your model.  For the former case, it is possible (in an "on paper" sense) to generate an initial history under the correct model, and doing so may require a custom coalescent simulation.  For the latter scenario, where no sensible definition of $N_e$ holds, you may simply have to simulate longer, and implement a check after simulating for a certain length of time that each marginal tree has a single root.
+
+We have only shown very simple examples of seeding with trees.  It is completely possible to seed a forward simulation with an `msprime.TreeSequence` generated under a model of population structure with unequal effective sizes, asymmetric migration, and change in effective sizes in each deme over time.  We leave that as an excercise for the reader...
+
+**Detail:** Readers familiar with `msprime` will note that, although the time units of the input `TreeSequence` are generations, that the input times are not all integer values.  That seems to be "fine", in the sense that we've done a lot of testing of simulation output and things are turning out as we expect them to.
```

Comment:
I don't really follow the point of this comment to be honest --- are you talking about the fact we're mixing times scaled back into generations from a coalescent simulation with integer generations from the forwards sim? Maybe rephrase or remove?